doc/man1/openssl-kdf.pod.in

```diff
@@ -93,8 +91,35 @@ The password must be specified for PBKDF2 and scrypt.
 
 =item B<hexpass:>I<string>
 
-Specifies the password in hexadecimal form (two hex digits per byte).
-The password must be specified for PBKDF2 and scrypt.
+Alternative to the B<pass:> option where
+the password is specified in hexadecimal form (two hex digits per byte).
+
+=item B<salt:>I<string>
+
+Specifies a non-secret random cryptographic salt as an alphanumeric string
```

Comment:
I worry about the use of `random` and `cryptographic` here.

First, with `random`, this might suggest it should be different each time it is used, as opposed to for each different key we want.

And secondly, while salts should be unique, unpredictability is "merely" "better" - being somewhat predictable (e.g. using a non-cryptographic random number generator) isn't as much of a deal-breaker as it is in the case of a symmetric key.

I wonder if we should use `unique` and `unpredictable` here, rather than `cryptographically random`?

Reply:
I'm not sure whether `random` or `unpredictable` is better here - what do others think?

Yet note that the proposed text does not say `cryptographically random`.
I deliberately chose `cryptographic salt` because this term is used on https://en.wikipedia.org/wiki/Key_derivation_function and I like it for distinction from regular salt (spice).
In the way I prepended 'cryptographic' to 'salt', it should be harmless and in IMHO somewhat helpful.

Reply:
`a non-secret unique cryptographic salt` would be my choice

Reply:
Done this now (in both documents).